README.md

```diff
@@ -4,115 +4,72 @@
-3.  Open _**cluster/custer.yaml**_ and update the following:
-    *   Attached ARN policies for the **alb-ingress-controller** and **external-dns**
-        *   format: ```arn:aws:iam::XXXXXXXXX:policy/AllowExternalDNSUpdates```
-    *   Public Key Path of public key used in your aws environment
-        *   [AWS KEY Pair Guide ](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html)
-        *   Steps:
-            1.  Create a key pair and output it into the pem file 
-                *  ```aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem```
-            2.  Generate the public key(pub) from the private key(pem)
-                * ```ssh-keygen -y -f MyKeyPair.pem > MyKeyPair.pub```
-            3.  Update the permisson of your pub file to only give read acess to the owner of the file
-                * ```chmod 400 MyKeyPair.pub```  
-            4. Pass the path to the public key file into cluster.yaml as the value for the **publicKeyPath**
-    *   Name and AWS Region of your eks cluster
-4.  Create the eks cluster
-    * ```eksctl create cluster -f cluster/cluster.yaml```
-    * This command will create the following:
-      * The EKS Cluster
-      * EKS Cluster role which is used to create aws resources for the kubernetes clusters
-      * Managed Node Groups that consists of EC2 Instances that are your worker nodes
-      * VPC, Public and private subnets
-      * Security groups for the control plane master nodes, load balancer, and communication between all nodes in the cluster
-      * IAM Roles for Service Accounts that allow cluster operators to map aws IAM roles to Kubernetes Service Account
-        * Current Service Accounts mapped:
-          * **alb-ingress-controller**
-          * **external-dns**
-        * NOTE: Use command below to show Roles created by EKSCTL with OIDC
-          * ```eksctl get iamserviceaccount --cluster cluster_name```
+1. You have a cluster and it has been setup properly
+2. You have helm and kubectl installed
+3. You have updated your kube-config to use the appropriate context
+   * To view contexts
+     * ```kubectl config get-contexts```
+   * To use different context
+     * ```kubectl config use-context {name of context}``` 
 
 ## Installation of Illumidesk Helm Chart 
 
 1. Verify that helm exists: ```helm list```
 2. Create a namespace for your helm chart
    * ```kubectl create namespace $NAMESPACE```
-3. Create a values yaml file locally and pass the chart values that you would like to override
-    * 
-    * You must override the following:
-    * | Key         | Description                                      | Command Line to get value  |
+3. Create a copy of _**example-config/values.yaml.example**_ file and update it with your setup. 
+    * NOTE to get a token use  ``` openssl rand -hex 32``` 
+    * Here is an example of a basic load balancer setup setup
+        ```bash 
+                jupyterhub:
```

Comment:
Instead of providing the user with an actual YAML copy, provide them with a table in a `Parameters` section. The YAML can change over time so having it in the Readme isn't DRY.

Reply:
Added this to TLDR